### PR TITLE
Migrate to Codemod@0.0.5

### LINKED ISSUE #5: Outdated Codemod version
The latest version of Codemod, <https://deno.land/x/codemod@0.0.4>, is released!

This calls for [`boardd/boardd.ts`](https://github.com/acmcsufoss/boardd/blob/e5e4d6757c524c1a154b8c77106cf73384a600ef/boardd/boardd.ts#L31) to be migrated to  Codemod@0.0.4.

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Migrate `boardd/boardd.ts` to [Codemod@0.0.5](https://deno.land/x/codemod@0.0.5). This version of Codemod enables us to safely define a sequence of GitHub operations in TypeScript.

Resolves #5.

### CODE REVIEW

File: deps.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-export { Ngrok } from "https://deno.land/x/ngrok@4.0.1/mod.ts";
-export type { GitHubCodemodBuilderCreatePROptions } from "https://deno.land/x/codemod@v0.0.3/github/mod.ts";
-export { GitHubCodemodBuilder } from "https://deno.land/x/codemod@v0.0.3/github/mod.ts";
-export type { GitHubAPIClientOptions } from "https://deno.land/x/codemod@v0.0.3/github/api/mod.ts";
-export type { Officer } from "https://acmcsuf.com/code/raw/65d3cac9653f4469f51a4dad224d8878795ffc55/src/lib/public/board/types.ts";
+export * as dotenv from "https://deno.land/std@0.191.0/dotenv/mod.ts";
+export * as discord from "https://deno.land/x/discord_api_types@0.37.43/v10.ts";
+export * from "https://deno.land/x/codemod@0.0.5/github/mod.ts";
+export type { GitHubAPIClientOptions } from "https://deno.land/x/codemod@0.0.5/github/api/mod.ts";
+// TODO: Update SHA to main permalink blocked by <https://acmcsuf.com/pull/870>. Try manual testing.
```
Comment (EthanThatOneKid) [inline deps.ts]: TODO(@EthanThatOneKid): Manual testing. Blocked by https://github.com/ethanthatonekid/acmcsuf.com/pull/870.

File: boardd/boardd.ts [hunk 2]
```diff
@@ -43,140 +39,204 @@ export async function boardd(options: BoarddOptions): Promise<BoarddResult> {
...
+              );
+          }
+
+          return tree.baseRef(ref);
+        })
+        .createCommit(({ 0: tree }) => ({
+          message: `Upload ${fullName}'s board member profile picture`,
+          tree: tree.sha,
+        }), (commit) => commit.parentRef(ref))
+        // TODO(https://oss.acmcsuf.com/codemod/issues/18): Update the current branch from main.
```
Comment (EthanThatOneKid) [inline boardd/boardd.ts:180]: TODO(@EthanThatOneKid): Manual testing. Blocked by https://github.com/acmcsufoss/codemod/issues/18.

File: deps.ts (hunk 1 above)
Comment (EthanThatOneKid) [reply deps.ts]: Resolved in <https://github.com/acmcsufoss/boardd/pull/6/commits/f12bd7ffdd4c1fe10346e9011db3a503776b59a8>.

**Review by EthanThatOneKid — COMMENTED** [review]
Manually tested in https://github.com/EthanThatOneKid/acmcsuf.com/pull/878. LGTM :godmode:

File: boardd/boardd.ts (hunk 2 above)
Comment (EthanThatOneKid) [reply boardd/boardd.ts:180]: [Resolved 😎](https://github.com/acmcsufoss/boardd/pull/6#pullrequestreview-1485182308).

**Comment by EthanThatOneKid** [comment]
Reminder(@EthanThatOneKid): Update <https://discord.com/developers/applications/1031627726454538281/information>'s "interaction endpoint URL" to <https://boardd.deno.dev> ([project](https://dash.deno.com/projects/boardd)).

**Review by k4rni — APPROVED** [review]
Excited to see it in action! Thank you for your hard work.

**Pull request merged**